.github/workflows/test.yml

```diff
@@ -47,7 +51,7 @@ jobs:
         openssl:
           - openssl-1.0.2u # EOL
           - openssl-1.1.0l # EOL
-          - openssl-1.1.1q
+          - openssl-1.1.1s
           - openssl-3.0.5
```

Comment:
Why not 3.0.7 too?

Reply:
One of those times when you distinctly decide to do something, but due to an interruption...

Sorry.  Fixed.